crates/sui-protocol-config/src/snapshots/sui_protocol_config__test__version_15.snap

```diff
@@ -21,6 +21,7 @@ feature_flags:
   narwhal_versioned_metadata: true
   zklogin_auth: true
   consensus_transaction_ordering: ByGasPrice
+  shared_object_deletion: true
```

Comment:
Is this just for the tests in the PR?

We won't be able to turn this on until we also have the core changes in (or we will need to add this to some future/far off protocol version). 
Just flagging :) Let me know what the plan is here!

Reply:
yes this is just for tests!